core/src/main/resources/defaults/blueprints/cdp-opdb-702.bp

```diff
@@ -74,6 +74,10 @@
           {
             "name": "hbase_wal_dir",
             "value": "/hbase-wals"
+          },
+          {
+            "name": "hbase_service_config_safety_valve",
+            "value": "<property><name>phoenix.schema.isNamespaceMappingEnabled</name><value>true</value></property><property><name>phoenix.queryserver.withRemoteUserExtractor</name><value>true</value></property><property><name>phoenix.functions.allowUserDefinedFunctions</name><value>true</value></property><property><name>hbase.use.dynamic.jars</name><value>false</value></property>"
```

Comment:
I'm assuming the safety valve is temporary while the CM work gets hashed out?

Reply:
The phoenix specific properties are here to stay, until we can hook in phoenix config properly into CM.

The dynamic JAR stuff is meant to be replaced with the proper CM property. It seems that in CloudBreak using safety valves is the norm, rather than the exception, so It's not a priority.

Reply:
Let's not make the problem worse. These configs if they are necessary should be exposed in CM. Safety valves are being removed where possible. https://jira.cloudera.com/browse/CB-4191